### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PatrickSJacobs**
### What are the issues this pull addresses (issue numbers / links)?

Improved README.md

### Did you add tests to cover your changes (yes/no)?

No (Seemingly not applicable)

### Did you update the documentation accordingly (yes/no)?

No (Seemingly not applicable)

### Did you read the CONTRIBUTING document (yes/no)?

Yes

### Summary

Improved README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,180 +1,95 @@
...
+While much of Qiskit is made up of elements that each work together to enable quantum computing, this element, Metal, is an open-source framework for engineers and scientists to design superconducting quantum devices with ease.
+## Installation
+We encourage installing Qiskit Metal via the pip tool (a python package manager).
+```bash
+pip install qiskit-metal
 ```
-
-Finally, you can also choose to build the documentation locally. To do so, from command line, change directory to `docs` and execute `python build_docs.py`.
-
-## Installation from the github repository
```
Comment (marcolincs) [inline README.md:38]: it seems the github-install instructions are removed. Can we instead move them to a separate `README_github.md`, or `README_contributor.md` or something like it? And maybe add a note here that is not much that we "encourage" them to install t this way, but more neutral: `to just use the tool, do this PyPI install`, and then add that `to edit the code and truly customize the experience, or if PyPI gives trouble, do the github install`......something like that.

File: README.md [hunk 2]
```diff
@@ -1,180 +1,95 @@
...
-**Qiskit Metal** is an open-source framework for designing quantum devices.
-
-
+# Qiskit Metal 
+[![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases)<!--- long-description-skip-begin -->[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg)](https://doi.org/10.5281/zenodo.4618153)
 >  Quantum hardware design and analysis
-
 ![Welcome to Qiskit Metal!](docs/images/zkm_banner.png 'Welcome to Qiskit Metal')
-
-### Quantum device design project
```
Comment (marcolincs) [inline README.md:13]: we need to keep these 4 sections (intro, recorded tut, live tut, slack). But you can compact them if you like, and maybe stash them into a title that sounds something like `Community and Support`.

**Review by marcolincs — CHANGES_REQUESTED** [review]
Great stuff! I did not check correctness of everything, but it does read nicely. I am heading out, so for now I document here a few comments/requests.

File: README.md [hunk 3]
```diff
@@ -1,180 +1,95 @@
...
-Use the slack channel.  Join [qiskit slack](https://ibm.co/joinqiskitslack) and then join the `#metal` channel to communicate with the developers and other participants.  You may also use this channel to inquire about collaborations.
-
-## Documentation
-Full code documentation can be found [here](https://qiskit.org/documentation/metal/)
-
-You can open the page also by executing:
-```python
-import qiskit_metal
-qiskit_metal.open_docs()
+Qiskit is an open-source framework for working with noisy quantum computers at the level of pulses, circuits, and algorithms.
```
Comment (marcolincs) [inline README.md:5]: There is no need to introduce Qiskit (in fact better not). Maybe replace this with the first of the 4 sections mentioned above?

**Review by marcolincs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JeremyDrysdale-ibm) [reply README.md:38]: Yes, the question, "what about developers" jumped out immediately at me.
- In the current readme there is a reference do `requirements-dev.txt`, which contains things like `pylint`, `yapf` among other things that developers will need to set up their environment.
- We also seem to have lost the `NEW_DEVELOPER_SETUP.md` instructions.

**Review by JeremyDrysdale-ibm — CHANGES_REQUESTED** [review]
See "what about developers" notes above.

File: README.md (hunk 2 above)
Comment (PatrickSJacobs) [reply README.md:13]: Done!

File: README.md (hunk 3 above)
Comment (PatrickSJacobs) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (PatrickSJacobs) [reply README.md:38]: Done!

File: README.md [hunk 4]
```diff
@@ -1,180 +1,95 @@
...
-If convenient, you can instead try to install directly in an existing conda environment `<env_name_exist>`. To do so, just replace the word `create` with `update` in the commands in the previous section. Find the resulting commands updated below for simplicity:
+Launch the Qiskit Metal GUI to interactively view, edit, and simulate a QDesign:
+```python
+>>> gui = MetalGUI(design)
 ```
-conda env update -n <env_name_exist> environment.yml
-conda activate <env_name_exist>
-python -m pip install --no-deps -e .
+A script is available [here](https://qiskit.org/documentation/metal/tut/overview/1.1%20High%20Level%20Demo%20of%20Qiskit%20Metal.html), where we also show the overview of Qiskit Metal.
+### Creating Your First Quantum Component (A Transmon Qubit) in Qiskit Metal:
```
[the remark this answers is not in the document]
Comment (PatrickSJacobs) [reply README.md]: Done!

File: README.md [hunk 5]
```diff
@@ -1,180 +1,113 @@
...
-
-## Documentation
-Full code documentation can be found [here](https://qiskit.org/documentation/metal/)
-
-You can open the page also by executing:
-```python
-import qiskit_metal
-qiskit_metal.open_docs()
+# Qiskit Metal 
+[![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases)<!--- long-description-skip-begin -->[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg)](https://doi.org/10.5281/zenodo.4618153)
```
Comment (zlatko-minev) [inline README.md:2]: Well, I'm a little bit sad to see the open-source and awesome icons go, but I suppose that they can be seen as more clutter than cute.

File: README.md [hunk 6]
```diff
@@ -1,180 +1,113 @@
...
 conda env create -n <env_name> environment.yml
 conda activate <env_name>
 python -m pip install --no-deps -e .
 ```
-Note the use of `--no-deps`. Indeed the `environment.yml` already instructs conda to install all the necessary package dependencies. We therefore prevent `setup.py` from overwriting them with the pip-equivalent packages, which might not be compatible with conda. 
 
-This creates a new environment with name `<env_name>` with all the necessary library dependencies.
-Then it activates the new environment.
-Finally installs the local Qiskit Metal code inside that environment.
+To edit the code and truly customize the experience, if PyPI gives you trouble, or to install from source, follow the instructions in the [documentation](https://qiskit.org/documentation/metal/installation.html) and/or the [installation instructions for developers](/README_developers.md).
```
Comment (zlatko-minev) [inline README.md:22]: Maybe adding a little bold heading here would be nice, to show this is the third option, it's a little bit easy to miss the developer option

File: README.md [hunk 7]
```diff
@@ -1,180 +1,113 @@
...
+```python
+>>> from qiskit_metal import designs, draw, MetalGUI, Dict, open_docs
+>>> design = designs.DesignPlanar()
+>>> design.overwrite_enabled = True
+>>> design.chips.main
+>>> design.chips.main.size.size_x = '11mm'
+>>> design.chips.main.size.size_y = '9mm'
+>>> gui = MetalGUI(design)
+```
+#### Launch the Qiskit Metal GUI to interactively view, edit, and simulate a QDesign:
```
Comment (zlatko-minev) [inline README.md:40]: These are great Xmas Mark I would suggest adding some images in here as well to make it more visual

File: README.md [hunk 8]
```diff
@@ -1,180 +1,113 @@
...
+>>> gui.main_window.close()
 ```
-You'll want to also use these instructions to [setup user environment](/docs/NEW_DEVELOPER_SETUP.md)
 
+A script is available [here](https://qiskit.org/documentation/metal/tut/overview/1.1%20High%20Level%20Demo%20of%20Qiskit%20Metal.html), where we also show the overview of Qiskit Metal.
 
+## Community and Support
 
-## Authors and Citation
+#### Watch the recorded tutorials
```
Comment (zlatko-minev) [inline README.md:79]: I'm a big fan of images and icons, so I was just adding a YouTube icon or image here

File: README.md [hunk 9]
```diff
@@ -1,180 +1,113 @@
...
+The streaming will also be recorded and made available [here](https://www.youtube.com/playlist?list=PLOFEBzvs-VvqHl5ZqVmhB_FcSqmLufsjb) for offline review.
 
-Qiskit Metal is the work of [many people](https://github.com/Qiskit/qiskit-metal/pulse/monthly) who contribute to the project at different levels. Metal was conceived and developed by [Zlatko Minev](zlatko-minev.com) at IBM; then co-led with Thomas McConkey. If you use Qiskit Metal, please cite as per the included [BibTeX file](https://github.com/Qiskit/qiskit-metal/blob/main/Qiskit_Metal.bib). For icon attributions, see [here](/qiskit_metal/_gui/_imgs/icon_attributions.txt).
+#### Take part in the live tutorials and discussion
+Through June 2021 we are offering live tutorials and Q&A. [Sign up](https://airtable.com/shrxQEgKqZCf319F3) to receive an invite to the upcoming sessions.  The streaming will also be recorded and made available for offline review.  Find [here](https://github.com/Qiskit/qiskit-metal/blob/main/README_Tutorials.md) more details on schedule and use the Slack channel to give us feedback and to request the most relevant content to you.
 
+#### Get help: Slack
```
Comment (zlatko-minev) [inline README.md:85]: I think it may be nice  if we had a slack icon here

File: README.md (hunk 5 above)
Comment (PatrickSJacobs) [reply README.md:2]: I can add them back! Do you want me to?

Comment (zlatko-minev) [reply README.md:2]: Consider the consequences of the two choices, and what each might lead to in terms of experience and subtext, then make a call. I'm good either way￼￼

File: README.md (hunk 9 above)
Comment (marcolincs) [reply README.md:87]: Something like this at the top of the page:
[join slack](https://ibm.co/joinqiskitslack)

and something like this here:
[join slack](https://ibm.co/joinqiskitslack)

**Review by marcolincs — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (PatrickSJacobs) [reply README.md:81]: Done, I misunderstood what Zlatko was saying.

File: README.md (hunk 9 above)
Comment (PatrickSJacobs) [reply README.md:87]: Thanks. Did not know how to make that one.

File: README_developers.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
+# Qiskit Metal For Developers
+
```
[the remark this answers is not in the document]
Comment (zlatko-minev) [reply README_developers.md:2]: Nice!

File: README.md [hunk 11]
```diff
@@ -1,180 +1,115 @@
...
-For other common installation issues, please refer to the [FAQ](https://qiskit.org/documentation/metal/faq.html)
-
-### Additional steps for developers
-
-If you are planning to develop the qiskit metal codebase, you need extra packages, which you can install by running the following command instead of (or after) the previous one:
+#### Launch the Qiskit Metal GUI to interactively view, edit, and simulate a QDesign:
+```python
+>>> gui = MetalGUI(design)
+```
+>![Example Gui #1!](docs/images/readme_gui_example_1.png 'Example Gui #1')
```
[the remark this answers is not in the document]
Comment (JeremyDrysdale-ibm) [reply README.md:44]: It looks like you deleted the images from the repo (readme_gui_example_1 and 3), but the refs are still present.

[the remark this answers is not in the document]
Comment (PatrickSJacobs) [reply README.md]: I believe this to be resolved.

[the remark this answers is not in the document]
Comment (PatrickSJacobs) [reply README.md]: Where are those references? I do not see them.

**Review by JeremyDrysdale-ibm — APPROVED** [review]

**Review by marcolincs — APPROVED** [review]
thank you for all the updates.
We already squashed, so I will merge as soon as Jeremy approves (and the checks pass)

**Review by JeremyDrysdale-ibm — APPROVED** [review]

**Pull request merged**